samples/demo-contract-framework/src/test/java/org/eclipse/dataspaceconnector/demo/contracts/DemoContractOfferFrameworkExtensionTest.java

```diff
@@ -48,6 +48,8 @@ void setUp(EdcExtension extension) {
     @Test
     void retrieveContractOffers(OkHttpClient httpClient, TypeManager typeManager) {
         ConnectorClient client = new ConnectorClient(httpClient, typeManager);
+        // for the 'id' we must actually use the constant from Asset, otherwise an exception gets raised during
```

Comment:
Is this comment needed? Otherwise we could add another test that shows the failure in the case of the absence of the correct id property

Reply:
good idea